encord/orm/dataset.py

```diff
@@ -120,14 +211,180 @@ def created_at(self, value: datetime) -> None:
         """Datetime will trim milliseconds for backwards compatibility."""
         self["created_at"] = value.strftime(DATETIME_STRING_FORMAT)
 
+    @property
+    def querier(self) -> Querier:
+        return self["querier"]
+
+    @querier.setter
+    def querier(self, new_querier: Querier) -> None:
+        self["querier"] = new_querier
+
+    @property
+    def frames_per_second(self) -> int:
+        if self.data_type != DataType.VIDEO:
+            return None
+        return self["frames_per_second"]
+
+    @frames_per_second.setter
+    def frames_per_second(self, new_frames_per_second: int) -> None:
+        self["frames_per_second"] = new_frames_per_second
+
+    @property
+    def duration(self) -> Optional[int]:
+        if self.data_type != DataType.VIDEO:
```

Comment:
these are great. Can we please add docstrings everywhere where it would make sense? For example, here you could say something like 
"""
Returns:
    If the data type is of type `DataType.VIDEO` this returns the duration. Otherwise it returns `None` as a duration field is not applicable.
"""
There will be other places where updating the doc strings will be necessary.